app/views/Dashboard/Country/styles.css

```diff
@@ -5,13 +5,12 @@
 
     .country-main {
         display: flex;
-        gap: var(--dui-spacing-extra-large);
+        gap: var(--dui-spacing-small);
 
         .country-detail-wrapper {
             display: flex;
-            flex-direction: column;
+            flex-flow: column;
```

Comment:
Why use flex-flow instead of flex-direction?

Reply:
Changes done